src/components/HubSpotForm.tsx

```diff
@@ -325,5 +330,10 @@ export const HubSpotForm: FunctionComponent<HubSpotFormProps> = ({
         }
     }, [formId, masterFormName, onFormSubmitted, inlineMessage, chiliPiper, formCreated])
 
-    return <div id="form-target" className={styles.container} />
+    return (
+        <div
+            id="form-target"
+            className={classNames(styles.container, overrideFormShorten && styles.overrideFormShorten)}
+        />
+    )
```

Comment:
`overrideFormShorten` css class and prop overrides `display: none` style rules which are made by ClearBit form shorten

![image](https://github.com/sourcegraph/about/assets/89894075/c76be2db-91dc-480e-8916-f2dfd8db333c)